easycv/toolkit/blade/cv_blade_utils.py

```diff
@@ -74,9 +74,9 @@ def opt_trt_config(input_config=dict(enable_fp16=True)):
         optimization_pipeline='TensorRT',
         enable_fp16=True,
         customize_op_black_list=[
-            'aten::select', 'aten::index', 'aten::slice', 'aten::view'
+            # 'aten::select', 'aten::index', 'aten::slice', 'aten::view', 'aten::upsample'
         ],
-        fp16_fallback_op_ratio=0.3,
+        fp16_fallback_op_ratio=0.05,
```

Comment:
why change to 0.05

Reply:
0.05 will be more efficiency for yolox export or most cv models. finally, we may need to support config this ratio by diff test result in blade export

Reply:
We level a  detai help doc in export doc and 0.05 is still accurate and fast for YOLOX  during our experments